docs/pages/includes/plugins/identity-export.mdx

```diff
@@ -148,4 +148,14 @@ You will refer to this file later when configuring the plugin.
   more practical certificate lifetime, e.g., `--ttl=8760h` to export a one-year
   certificate.
 
+  Note that you cannot issue certificates that are valid longer than your existing credentials.
+  For example, to issue certificates with a 1000h TTL, you must be logged in with a session that is
```

Comment:
```suggestion
  For example, to issue certificates with a 1000-hour TTL, you must be logged in with a session that is
```
I thought it would flow more easily to use a word here, rather than the Go duration syntax